src/unitxt/metrics.py

```diff
@@ -1412,6 +1416,13 @@ def verify(self):
 
     def prepare(self):
         super().prepare()
+        if (
+            hasattr(self, "postpreprocess_steps")
+            and self.postpreprocess_steps is not None
+            and isinstance(self.postpreprocess_steps, list)
+            and len(self.postpreprocess_steps) > 0
+        ):
+            self.postprocess_steps = self.postpreprocess_steps
```

Comment:
Need to check that self.postproces_step is not set (throw error if both are set).

Reply:
Done.